qiita_db/metadata_template.py

```diff
@@ -1590,6 +1590,25 @@ def create(cls, md_template, raw_data, study, data_type,
             conn_handler.execute(
                 "DELETE FROM qiita.prep_template where "
                 "{0} = %s".format(cls._id_column), (prep_id,))
+
```

Comment:
This whole new patch lives inside a `try/except Exception`, is there any way to avoid this?